station/submitobs.py

```diff
@@ -96,7 +97,17 @@ def submit_observation(data: SubmitRequestData):
         )
     }
 
-    requests.post(url, form_data, headers=headers, files=files)
+    logging.info("Submitting observation, url=%s file=%s" % (url, filename))
+
+    resp = requests.post(url, form_data, headers=headers, files=files)
+    if (resp.status_code >= 400):
+        logging.warn("Response status: %d" % resp.status_code)
+    else:
+        logging.info("Response status: %d" % resp.status_code)
+    if (resp.status_code != 200):
+        logging.info("Submission details: headers=%s" % headers)
+        logging.info("form=%s" % form_data)
```

Comment:
Is it a big message? Form data contains a lot of data and binaries. Maybe it should be the "DEBUG" level?

Reply:
form_data is around 200-250 bytes (aos, tca, los, sat name, notes, rating and tle).